LayoutTests/editing/execCommand/canvas-counter-rooteditable-crash-expected.txt

```diff
@@ -0,0 +1,8 @@
+layer at (0,0) size 800x600
```

Comment:
Please call testRunner.dumpAsText so that the output will be plain text.

Reply:
Done. :-) Thanks for your guidance, learnt something new. :-)